ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -97,6 +98,14 @@ function redirectToNiH(): void {
   window.open(url, '_blank');
 }
 
+function redirectToRas(): void {
+  AnalyticsTracker.Registration.RasLoginGov();
+  const url = serverConfigStore.getValue().rasHost + '/auth/oauth/v2/authorize?client_id=' + serverConfigStore.getValue().rasClientId
+      + '&prompt=login+consent&redirect_uri=' + buildRasRedirectUrl()
+      + '&response_type=code&scope=openid+profile+email+ga4gh_passport_v1';
```

Comment:
Do we need to keep this synchronized with the scope list on the server? If so, please add a comment

Reply:
add some comment, but actually userInfo fetching in BE, is optional(or a bug in RAS, I can use any scope there)